omod/src/test/java/org/openmrs/module/idgen/rest/search/LogEntrySearchHandlerTest.java

```diff
@@ -82,7 +82,78 @@ public void getSearchConfig_shouldReturnLogEntryByDateGenerated() throws Excepti
 	public void getSearchConfig_shouldReturnLogEntryByGeneratedBy() throws Exception {
 		MockHttpServletRequest req = request(RequestMethod.GET, getURI());
 		req.addParameter("generatedBy", USER_UUID);
+		req.addParameter("comment", "New");
```

Comment:
The package of this class does not match its folder structure. Can you look into that?

Reply:
The same goes for LogEntryResourceTest

Reply:
Changed the packages. Its because I moved the both tests from the web to rest folder and I forgot to update the packages accordingly